### ISSUE TITLE
About multi-modal

### ISSUE THREAD

**Issue by AlexNmSED** [wisdomikezogwo/MMAE_Pathology#4]
Hello, I want to run extractHE. But I can't find the scale_csv.csv for NCT data. Can you help me to do this ?

Thank you.

**Comment by AlexNmSED** [comment]
And I need mae_patches.csv too.  Can you provide it?

**Comment by wisdomikezogwo** [comment]
Apologies for the delay in my reply, I'll look into this shortly

**Comment by wisdomikezogwo** [comment]
The files you request are for WSI image-level pretraining and not patch-level image pertaining like with the patch-level NCT dataset. because the code is setup to handle both its a bit confusing I admit, however you can choose your dataset as an arguments to extractHE and wont require any of these files as they would be automatically created if your dataset has been downloaded properly in the right directory.

So, few steps:
1. please ensure you have download the dataset, such that this line in extractHE points to it correctly.
`mae_img_path = os.path.join(PROCESSED_BASE_DIR, 'pyramid', f"{args.dataset}", f'{train_or_test}', 'mae')`

2. `python -m extractHE --dataset NCK -nmil --img_size 224 --n_workers 12 `

Please let me know if you have any other issues!

**Comment by AlexNmSED** [comment]
Thank you for your help!

After pre-training, can I obtain the features of each 16*16 image patch for subsequent operations. (sorry, I'm a newbie)

**Comment by wisdomikezogwo** [comment]
Yes you can, from any layer of the transformer. That should be all other tokens except the CLS token.
